ethsigner/commandline/build.gradle

```diff
@@ -0,0 +1,44 @@
+
+jar {
+  baseName 'ethsigner-commandline'
+  manifest {
+    attributes(
+      'Specification-Title': baseName,
+      'Specification-Version': project.version,
+      'Implementation-Title': baseName,
+      'Implementation-Version': calculateVersion()
+      )
+  }
+}
+
+
+dependencies {
+
+  implementation project(':ethsigner:signing-api')
+  implementation project(':ethsigner:core')
+
+  implementation 'info.picocli:picocli'
+  implementation 'com.google.guava:guava'
+
+  implementation 'org.apache.logging.log4j:log4j-api'
+  runtime 'org.apache.logging.log4j:log4j-core'
+  runtime 'org.apache.logging.log4j:log4j-slf4j-impl'
+
+  implementation 'org.web3j:core'
+  implementation 'org.web3j:crypto'
+  implementation 'org.web3j:eea'
+
+  implementation 'io.vertx:vertx-core'
+  implementation 'io.vertx:vertx-web'
+  implementation 'io.vertx:vertx-web-client'
+
+  testCompile 'io.vertx:vertx-codegen'
```

Comment:
This should either be `testImplementation` (the replacement for testCompile) or `compileOnly`, depending exactly on what you're wanting.

Reply:
actually - it just needs to be removed _almost_ across the board.